Doc/library/importlib.metadata.rst

```diff
@@ -238,8 +238,6 @@ Python packages or modules::
     >>> packages_distributions()
     {'importlib_metadata': ['importlib-metadata'], 'yaml': ['PyYAML'], 'jaraco': ['jaraco.classes', 'jaraco.functools'], ...}
 
-.. versionadded:: 3.10
```

Comment:
I guess this docs change to remove the directive can be reverted since this was added in Python 3.10

Reply:
Oh, good catch. Yes, indeed!

Reply:
Fixed in 3063cf31d2.